Makefile

```diff
@@ -1,4 +1,5 @@
 FILE=./VERSION
 RESUMIC_VERSION = `cat $(FILE)`
+GITCOMMIT = `git describe --tags --always --dirty`
 build: ./cmd/version.go
```

Comment:
Adding a space between setting the variables and the build cod makes the code easier to read.